pyof/foundation/network_types.py

```diff
@@ -3,14 +3,48 @@
 Defines and Implements Basic Network packet types , such as Ethertnet and LLDP.
 """
 
+# System imports
+from enum import IntEnum
+
+# Local source tree imports
 from pyof.foundation.base import GenericStruct
 from pyof.foundation.basic_types import (
     BinaryData, HWAddress, IPAddress, UBInt8, UBInt16)
-from pyof.foundation.constants import VLAN_TPID
 from pyof.foundation.exceptions import PackException, UnpackException
 
-__all__ = ('ARP', 'Ethernet', 'GenericTLV', 'IPv4', 'VLAN', 'TLVWithSubType',
-           'LLDP')
+__all__ = ('ARP', 'Ethernet', 'EtherType', 'GenericTLV', 'IPv4', 'VLAN',
+           'TLVWithSubType', 'LLDP')
+
+
+# NETWORK CONSTANTS AND ENUMS
+class EtherType(IntEnum):
```

Comment:
I think we can add the MPLS unicast/multicast values (0x8847/0x8848),

Reply:
Done! =)